### ISSUE TITLE
UMT localization

### ISSUE THREAD

**Issue by sahlaysta** [krzys-h/UndertaleModTool#181]
I can localize the interface to Spanish

**Comment by Mc-muffin** [comment]
Well, there's an UI rework planned so it might be wise to wait for that, also I don't think there's any localization code right now so that would need to be implemented first (unless you want to make a distinct localized version)

**Comment by colinator27** [comment]
Yeah, localization will need to wait for a rewrite. I don't think WPF and the current project are setup for that kind of thing at all.

**Comment by colinator27** [comment]
If/when that does happen, though, we could try some kind of JSON file or something with all of the strings used in the UI, or maybe another file because JSON is weird with commas and Git.

**Comment by Kneesnap** [comment]
Ooh. This is totally something we should do with the new UI, have support for different languages.

**Comment by Kneesnap** [comment]
I have included this in the roadmap at https://github.com/krzys-h/UndertaleModTool/issues/182
